invenio_app_ils/records/views.py

```diff
@@ -58,6 +60,15 @@ class DocumentStatsResource(ContentNegotiatedMethodView):
     @pass_record
     def post(self, pid, record, **kwargs):
         """Send a signal to count record view for the record stats."""
+        if not is_public(record):
+            permission = backoffice_permission()
+            if permission is not None and not permission.can():
+                if not current_user.is_authenticated:
+                    abort(401)
+                abort(403)
+
+        if record.pid.pid_type != "docid" and record.pid.pid_type != "serid":
```

Comment:
actually this is not needed: above, the view is registered for:
```
register_view(DOCUMENT_PID_TYPE)
register_view(EITEM_PID_TYPE)
```
so it won't work in other urls.
You can remove this check, and change above to:
```
register_view(DOCUMENT_PID_TYPE)
register_view(SERIES_PID_TYPE)
register_view(EITEM_PID_TYPE)
```

The UI unfortunately is not showing views stats for series, but maybe we add it...